src/xunit.performance.run/ProgramCore.cs

```diff
@@ -137,11 +141,25 @@ private void RunTests(XunitPerformanceProject project)
             };
 
             startInfo.Environment["XUNIT_PERFORMANCE_RUN_ID"] = project.RunId;
-            startInfo.Environment["XUNIT_PERFORMANCE_MIN_ITERATION"] = "10";
-            startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION"] = "1000";
-            startInfo.Environment["XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS"] = "1000";
-            startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
-            startInfo.Environment["COMPLUS_gcServer"] = "0";
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MIN_ITERATION")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MIN_ITERATION"] = MinIterations.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_ITERATION")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION"] = MaxIterations.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_ITERATION_INNER_SPECIFIED")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_ITERATION_INNER_SPECIFIED"] = MaxIterationsWhenInnerIterationsSpecified.ToString();
+
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS")))
+                startInfo.Environment["XUNIT_PERFORMANCE_MAX_TOTAL_MILLISECONDS"] = MaxTotalMSPerTest.ToString();
+
+            // Disable server and background GC.
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("COMPLUS_gcConcurrent")))
+                startInfo.Environment["COMPLUS_gcConcurrent"] = "0";
```

Comment:
Concurrent GC and server GC were both disabled prior to this change.  What I've done here is allow them to be overridden. I could definitely believe that concurrent GC would preturb some numbers, especially for instructions retired.  I completely agree though that you want to be able to control these, especially for GC tests - thus the ability to override them now.

I did not experiment with removing these.  I believe that these were added by @ericeil who likely has more context.